example/src/shared/Wysiwyg/WysiwygNav.js

```diff
@@ -24,16 +25,26 @@ import {
   MainButtons,
   CustomIconButton,
   NavWrapper,
-  MoreButton
+  MoreButton,
+  IconButtonGroupMargin
 } from './WysiwygStyles';
 
-const WysiwygNav = ({ placeholder, onActionClick }) => {
+const WysiwygNav = ({ 
```

Comment:
In this component, you do 13 times the condition `isPreviewMode ? true : false`. Also, you don't event need to render the popover on preview mode.

You can do the condition only one time before the render:
```jsx
const WysiwygNav = ({ ..., isPreviewMode }) => {
   const buttonMoreRef = useRef();

   if (isPreviewMode) {
       return (
           <NavWrapper padding={2} background="neutral100">
              <Row justifyContent="space-between">
                <Row>
                  <Select
                    disabled
                    id="selectTitle"
                    placeholder={placeholder}
                    size="S"
                    onChange={(value) => onActionClick(value)}
                  >
                    <Option value="h1">h1</Option>
                    <Option value="h2">h2</Option>
                    <Option value="h3">h3</Option>
                    <Option value="h4">h4</Option>
                    <Option value="h5">h5</Option>
                    <Option value="h6">h6</Option>
                  </Select>
      
                  <MainButtons>
                    <CustomIconButton disabled id="Bold" label="Bold" name="Bold" icon={<Bold />} />
                    <CustomIconButton disabled id="Italic" label="Italic" name="Italic" icon={<Italic />} />
                    <CustomIconButton disabled id="Underline" label="Underline" name="Underline" icon={<Underline />} />
                  </MainButtons>
      
                  <MoreButton disabled id="more" label="more" icon={<More />} />
                </Row>
      
                <Button onClick={() => setIsPreviewMode((prev) => !prev)} variant="tertiary" size="L" id="preview">
                  Markdown mode
                </Button>
              </Row>
            </NavWrapper>
       );
   }

   return (
      // render the editable mode :)
   )
}
```